libnetwork/drivers/bridge/setup_bridgenetfiltering.go

```diff
@@ -31,17 +31,15 @@ func getIPVersion(config *networkConfiguration) ipVersion {
 	return ipVer
 }
 
-func setupBridgeNetFiltering(config *networkConfiguration, i *bridgeInterface) error {
-	err := checkBridgeNetFiltering(config, i)
-	if err != nil {
+func setupBridgeNetFiltering(config *networkConfiguration, _ *bridgeInterface) error {
+	if err := checkBridgeNetFiltering(config); err != nil {
 		if ptherr, ok := err.(*os.PathError); ok {
 			if errno, ok := ptherr.Err.(syscall.Errno); ok && errno == syscall.ENOENT {
```

Comment:
That's a very old-fashioned way of writing
```go
if errors.Is(err, syscall.ENOENT) {
```
or, if you want to be really pedantic about only matching an errno when it is wrapped in a PathError:
```go
var ptherr *os.PathError
if errors.As(err, &ptherr) && errors.Is(ptherr, syscall.ENOENT) {
```

Reply:
OHMAN, yes. I may include that one in my follow-up 😅